martin/src/bin/martin-cp.rs

```diff
@@ -392,7 +392,7 @@ fn parse_encoding(encoding: &str) -> MartinCpResult<AcceptEncoding> {
 async fn init_schema(
     mbt: &Mbtiles,
     conn: &mut SqliteConnection,
-    sources: &[&dyn Source],
+    sources: &[Box<dyn Source>],
```

Comment:
I'm still not certain why you need a boxed type here - what caused it to change?  Also, you might want to use `TileInfoSource` for consistency